src/build_wrappers.js

```diff
@@ -0,0 +1,45 @@
+const fs = require('fs');
+const path = require('path');
+const { execSync } = require('child_process');
+const globby = require('globby');
+const { DIST_DIR_NAME, WRAPPERS_DIR_NAME } = require('./consts');
+
+/**
+ * Creates zips of all wrappers
+ */
+exports.buildWrappers = async function () {
```

Comment:
It would be good to have this logic shared somehow across with the `build_templates.js` script.